problem.schema.json

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
I haven't taken a look at this yet, but it seems to be missing `ADD` and `MUL` etc